go/common/types.go

```diff
@@ -78,3 +80,48 @@ type (
 	EncryptedSharedEnclaveSecret []byte
 	EncodedAttestationReport     []byte
 )
+
+type BlockAndReceipts struct {
```

Comment:
this needs a bit more clarification.

- Who is supposed to calculate the receipts? Is it the ethereum node? Is it the enclave?
- Does this structure need all receipts? Shouldn't it be enough to contain the relevant receipts?
- If the receipts originate externally, it needs to do some checking